### DISCUSSION TITLE
NAS的Docker从Alist发送下载总是提示Failed to send to aria2: AxiosError: Network Error

### DISCUSSION THREAD

**Discussion by iorangy** [AlistGo/alist#3716] [Q&A]
使用Alist发送下载到Aria2总是提示发送失败，是什么情况啊？NAS是绿联DX4600，安装时提示本地6800端口被占用（不知道是不是自带离线下载占用的），于是将6800的本地端口设置成了8开头四位数，不知道是不是跟这个有关。

**Comment by iorangy** [comment]
更新：原来在主页也需要设置Aria2端口

**Reply by Dtwangad** [reply]
怎么在主页设置Aria2端口啊

**Reply by iorangy** [reply]
主页右下角有一个齿轮

**Comment by ConcreteW** [comment]
感谢楼主就我一命， 不懂在aria2设置里改了地址了， 首页为啥还要设置 ，折腾了半天
